source/simulator/parameters.cc

```diff
@@ -657,6 +658,40 @@ namespace aspect
                            "is largely empirically decided -- it must be large enough to ensure "
                            "the bilinear form is coercive, but not so large as to penalize "
                            "discontinuity at all costs.");
+        prm.declare_entry ("Use limiter for discontinuous temperature solution", "false",
+                           Patterns::Bool (),
+                           "Whether to apply the bound preserving limiter as a correction after having"
+                           "the discontinous temperature solution. Currently we apply this only to the"
+                           "temperature solution if the global max/min are already determined and defined"
```

Comment:
What does "determined and defined" mean here? Can you explain? (I suspect that you are referring to another parameter, maybe reference it by name.)